include/nbl/asset/interchange/SAssetBundle.h

```diff
@@ -109,6 +109,10 @@ class SAssetBundle
 			m_contents->operator[](offset) = std::move(_asset);
 			assert(allSameTypeAndNotNull());
 		}
+		inline core::SRange<core::smart_refctd_ptr<IAsset>> getMutableContents()
```

Comment:
why would we even have this?

Reply:
ah ok I see private scope

Reply:
`getMutableContentPointers` more appropriate, we can always mutate content assets anyway from the `getContents() const`

Reply:
oh, right, via `smart_refctd_ptr::get()` yea, then this is not needed. 
Actually this is a little inconsistent since `operator->() const` returns `const T*` and `get() const` returns `T*`

Reply:
yeah its a bit weird about the `smart_refctd_ptr::operator->() const` returning `const T*`

should probably be fixed